docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSample.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.AnomalyDetection
+{
+    public static class RandomizedPcaSample
+    {
+        /// <summary>
+        /// Example with 3 feature values.
+        /// </summary>
```

Comment:
please don;t use XML style comments in samples, as it becomes harder to read than simple comments.

Reply:
My bad. Thanks.

---
In reply to: [261047221](https://github.com/dotnet/machinelearning/pull/2780#discussion_r261047221) [](ancestors = 261047221)